Update exporter README.md (#564)
open-telemetry/opentelemetry-collector: exporter/README.md

# General Information

An exporter is how data gets sent to different systems/back-ends. Generally, an

> exporter translate the internal format into another defined format.

Reviewer: ```suggestion
exporter translates the internal format into another defined format and sends the data elsewhere (typically via network).
```

Supported trace exporters (sorted alphabetically):

- [Jaeger](#jaeger)
- [OpenCensus](#opencensus)
- [Zipkin](#zipkin)

Supported metric exporters (sorted alphabetically):

- [OpenCensus](#opencensus)
- [Prometheus](#prometheus)

Supported local exporters (sorted alphabetically):

> - [Logging](#logging)

Reviewer: Also add File Exporter.

Author: Done

# General Information

An exporter is how data gets sent to different systems/back-ends. Generally, an
exporter translates the internal format into another defined format.

Supported trace exporters (sorted alphabetically):

- [Jaeger](#jaeger)
- [OpenCensus](#opencensus)
- [Zipkin](#zipkin)

Supported metric exporters (sorted alphabetically):

- [OpenCensus](#opencensus)
- [Prometheus](#prometheus)

Supported local exporters (sorted alphabetically):

- [File](#file)
- [Logging](#logging)

> The [contributors repository](https://github.com/open-telemetry/opentelemetry-service-contrib)

Reviewer 2: ```suggestion
The [contributors repository](https://github.com/open-telemetry/opentelemetry-collector-contrib)
```

>  has more exporters that can be added to custom builds of the service.

Reviewer 2: ```suggestion
 has more exporters that can be added to custom builds of the Collector.
```
